core/controllers/library.py

```diff
@@ -73,6 +73,12 @@ class LibraryPage(base.BaseHandler):
     def get(self):
         """Handles GET requests."""
         search_mode = 'search' in self.request.url
+        rank_mode = 'explorations' in self.request.url
+
+        if search_mode:
```

Comment:
So for now do I leave the title to be 'Find explorations to learn from - Oppia' for the top rated and recently published pages?